test/ck_tile/gemm/test_gemm_mem_pipeline.cpp

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2024, Advanced Micro Devices, Inc. All rights reserved.
+
+#include <tuple>
+
+#include "gtest/gtest.h"
+
+#include "ck_tile/host.hpp"
+#include "test_gemm_mem_pipeline_util.hpp"
+
+using F16 = ck_tile::half_t;
+using F32 = float;
+
+using Row = ck_tile::tensor_layout::gemm::RowMajor;
+using Col = ck_tile::tensor_layout::gemm::ColumnMajor;
+
+// clang-format off
+using KernelTypes = ::testing::Types<
+    //         ALayout, BLayout, CLayout, ADataType, BDataType, AccDataType, CDataType
+    std::tuple<    Row,     Col,     Row,       F16,       F16,         F32,      F32>
+    // TODO: fixme!
+    // std::tuple<    Col,     Row,     Row,       F16,       F16,         F32,      F32>,
+    // std::tuple<    Row,     Row,     Row,       F16,       F16,         F32,      F32>,
+    // std::tuple<    Col,     Col,     Row,       F16,       F16,         F32,      F32>
+    >;
+// clang-format on
+
```

Comment:
why disabled?

Reply:
They're not working :) Need to fix in future PR